react/features/conference/components/web/Conference.js

```diff
@@ -234,7 +242,7 @@ class Conference extends AbstractConference<Props, *> {
 
                     { _showPrejoin || _showLobby || <Toolbox /> }
 
-                    {_notificationsVisible && (_overflowDrawer
+                    {_notificationsVisible && !_isAnyOverlayVisible && (_overflowDrawer
```

Comment:
Why is this necessary now?

Reply:
So notifications don't display overlay the overlays.